lms/templates/edxnotes/note-item.underscore

```diff
@@ -17,7 +17,12 @@
     <ol class="note-comments" role="region" aria-label="<%- gettext('Note') %>">
         <li class="note-comment">
             <p class="note-comment-title"><%- gettext("You commented...") %></p>
-            <p class="note-comment-p"><%= text %></p>
+            <p class="note-comment-p">
+                <%= interpolate_text(_.escape(text), {
+                    highlighted_notes_start: '<span class="note-highlight">',
```

Comment:
@muzaffaryousaf I am continuing the conversation to help finalize a best practice.  Please ignore and keep this code as-is.

@andy-armstrong The "${tag_name}" formatting comes from your proposal in the [underscore/i18n JIRA story](https://openedx.atlassian.net/browse/TNL-3966?focusedCommentId=164494&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-164494).  To ease the burden on the transition, including updating translation strings, should we switch the proposal back to "{tag_name}" and drop the $?